src/foam/core/ticket/TicketComment.js

```diff
@@ -52,7 +52,7 @@ foam.CLASS({
     {
       class: 'String',
       name: 'comment',
-      label: '',
+      columnPermissionRequired: true,
```

Comment:
I think we want to keep the empty label since thats for the detail view. The code you want is this. I am also not sure if we need column permission for this, why would we?

```suggestion
      label: '',
      columnLabel: 'Comment'
```